src/bmp/BMPParser.h

```diff
@@ -10,10 +10,9 @@
 #include <string>
 #include <cassert>
 
-using namespace std;
-
 namespace BMPParser{
   typedef uint8_t byte;
+  typedef std::string string;

```

Comment:
Ah, I didn't see the `uint8_t byte` typedef in my initial review; I thought you were using `std::byte`.

This string typedef now creates an ambiguous symbol (`BMPParser::string` vs. `std::string` -- compilation error with MSVC but not GCC apparently).

Do you need either typedef? It'd be clearer without them I think, and avoid the above bug.

Reply:
Removed both typedefs.
But that is weird, I compiled with the latest msvc and didn't get even a warning.